### ISSUE TITLE
some fonts crash fontbakery with: ot-sanitise returned non-zero exit status 1

### ISSUE THREAD

**Issue by felipesanches** [googlefonts/fontbakery#817]
NATS-Regular.ttf:

```
DEBUG    Check #25: Checking with ot-sanitise  
Traceback (most recent call last):
  File "./fontbakery-check-ttf.py", line 2590, in <module>
    main()
  File "./fontbakery-check-ttf.py", line 1172, in main
    stderr=subprocess.STDOUT)
  File "/usr/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['ot-sanitise', '/home/felipe/devel/github_google/fonts/ofl/nats/NATS-Regular.ttf']' returned non-zero exit status 1
```

**Comment by felipesanches** [comment]
Same issue with "github_google/fonts/ofl/seoulhangang/SeoulHangang-Bold.ttf":

```
DEBUG    Check #21: Checking with ot-sanitise  
Traceback (most recent call last):
  File "./fontbakery-check-ttf.py", line 2692, in <module>
    main()
  File "./fontbakery-check-ttf.py", line 1176, in main
    stderr=subprocess.STDOUT)
  File "/usr/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['ot-sanitise', '/home/felipe/devel/github_google/fonts/ofl/seoulhangang/SeoulHangang-Bold.ttf']' returned non-zero exit status 1

```

**Comment by felipesanches** [comment]
Found another instance. So let's make a list:

* ofl/seoulnamsan/SeoulNamsan-Bold.ttf
* ofl/seoulhangang/SeoulHangang-Bold.ttf
* ofl/nats/NATS-Regular.ttf

**Comment by felipesanches** [comment]
These 3 fonts have been temporarily blacklisted on the ot-sanitize check at https://github.com/googlefonts/fontbakery/commit/fa8a6cdf18f54c5940692fcfc191fe911c2e3d6f

**Comment by felipesanches** [comment]
Removed the blacklisting and fixed the issue at a135454b2610dc9b4b1fa46df233b115ccf606d2
